50-config.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+# Set omero config properties from CONFIG_ envvars
+# Variable names should replace "." with "_" and "_" with "__"
```

Comment:
Not particularly worried but there's obviously some danger of a `__` conflict here.